packages/xo-web/src/xo-app/settings/config/xo-cloud-config/index.js

```diff
@@ -0,0 +1,131 @@
+          if (isConfigDefined) {
+            window.open(config.content, '_blank')
+          }
+        },
+      uploadCloudXoConfig:
+        () =>
+        async ({ config, isConfigDefined }) => {
+          if (isConfigDefined) {
+            const resp = await fetch(`./rest/v0/cloud/xo-config/backups/${config.id}/actions/import?sync`, {
+              method: 'POST',
+            })
+            if (!resp.ok) {
+              throw new Error(resp.statusText)
+            }
+            return {
+              config: undefined,
+            }
+          }
+        },
+      onChangeCloudXoConfig: (_, config) => ({
+        config,
+      }),
+      toggleEnableCloudXoConfig:
+        () =>
+        async (state, { cloudXoConfig }) => {
+          let passphrase
+          if (!cloudXoConfig.enabled) {
+            const params = await confirm({
+              icon: 'backup',
+              title: _('cloudXoConfig'),
+              body: <BackupXoConfigModal />,
+            })
+            passphrase = params.passphrase
+          }
+
+          const resp = await fetch('./rest/v0/cloud/xo-config', {
+            method: 'PATCH',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+              enabled: !cloudXoConfig.enabled,
+              passphrase,
+            }),
+          })
+          if (!resp.ok) {
+            throw new Error(resp.statusText)
+          }
+          subscribeCloudXoConfig.forceRefresh()
+        },
+    },
+    computed: {
+      applianceId: async () => {
+        const { id } = await getApiApplianceInfo()
+        return id
+      },
+      groupedConfigs: ({ applianceId, sortedConfigs }) =>
+        sortBy(groupBy(sortedConfigs, 'xoa_id'), config => (config[0].xoa_id === applianceId ? -1 : 1)),
+      isConfigDefined: ({ config }) => config != null,
+      sortedConfigs: (_, { cloudXoConfigBackups }) =>
+        cloudXoConfigBackups?.sort((config, nextConfig) => config.created_at - nextConfig.created_at),
+    },
+  }),
+  injectState,
+  ({ effects, state, cloudXoConfig }) => (
+    <div>
+      <div className='mb-1'>
+        <em>
+          <Icon icon='info' /> {_('xoConfigCloudBackupTips')}
+        </em>
+        <br />
+        <ActionButton
+          btnStyle={cloudXoConfig?.enabled ? 'warning' : 'primary'}
+          handler={effects.toggleEnableCloudXoConfig}
+          icon='backup'
+        >
+          {cloudXoConfig?.enabled ? _('disableXoCloudBackup') : _('enableXoCloudBackup')}
+        </ActionButton>
+      </div>
+      <div>
+        <h2>{_('xoConfig')}</h2>
+        <SelectXoCloudConfig onChange={effects.onChangeCloudXoConfig} value={state.config} />
+        <div className='mt-1'>
+          <ActionButton
+            handler={effects.uploadCloudXoConfig}
+            btnStyle='warning'
+            icon='upload'
+            disabled={!state.isConfigDefined}
+          >
+            {_('importXoConfig')}
```

Comment:
I'm not sure if we should call it "Backup current XO Config" even if it's a one time backup?
/cc @olivierlambert

Reply:
Oh, I thought it was used to trigger one backup of the current config. But then let's simply call it... "restore"!

(and there's no way to trigger a one time backup, right?)

Reply:
@julien-f was not fan of `restore` since it doesn't replace the current config.
No.

Reply:
I'm open to a better idea.

Reply:
After discussion, let's use "restore" for now. We'll see if users aren't too confused by the fact the config gets merged into the old one and we might change it in the future and/or in XO 6.

Main rationale: homogeneous with the VM vocabulary.
- XO ↔ local: import/export
- XO ↔ cloud: backup/restore